src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -1817,12 +1817,14 @@ End Module</Document>)
             End Using
         End Function
 
+        ' The test verifies the CommitCommandHandler isolated behavior which does not add '()' after 'Main'.
+        ' The integrated VS behavior for the case is to get 'Main()'.
         <WorkItem(792569, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/792569")>
         <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function CommitOnEnter(completionImplementation As CompletionImplementation) As Task
             Dim expected = <Document>Module M
     Sub Main()
-        Main()
```

Comment:
>() [](start = 12, length = 2)

This reverts the temportary fix: [https://github.com/dotnet/roslyn/pull/27027/files#diff-fe605b679d28134d144ac985d235187aR1811](https://github.com/dotnet/roslyn/pull/27027/files#diff-fe605b679d28134d144ac985d235187aR1811)